privacyidea/webui/login.py

```diff
@@ -45,10 +45,10 @@
 from privacyidea.lib.queue import has_job_queue
 
 DEFAULT_THEME = "/static/contrib/css/bootstrap-theme.css"
+DEFAULT_LANGUAGE_LIST = ["en", "de", "de_DE", "nl", "fr",
+                         "cs", "es", "tr", "zh_Hant"]
```

Comment:
According to our weblate translations the languages with more than 50% translations are:
`['en', 'de', 'nl', 'zh_Hant', 'fr', 'es', 'tr']`

Reply:
fixed in 09d11230e87bdf58510ca5d85aed0f8271538b71.